longevity_operator_multi_tenant_test.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2022 ScyllaDB
+
+import copy
+
+from longevity_test import LongevityTest
+
+from sdcm.utils.common import ParallelObject
+
+
+# pylint: disable=too-many-instance-attributes
+class ScyllaClusterStats(LongevityTest):
+    # pylint: disable=too-many-arguments,super-init-not-called
+    def __init__(self, db_cluster, loaders, monitors, prometheus_db, params, test_config, cluster_index):
+        self.db_cluster = db_cluster
+        self.loaders = loaders
+        self.monitors = monitors
+        self.prometheus_db = prometheus_db
+        self.params = copy.deepcopy(params)
+        self._es_doc_type = "test_stats"
+        self._stats = self._init_stats()
+        self.test_config = test_config
+        self._duration = self.params.get(key='test_duration')
+        self.create_stats = self.params.get(key='store_perf_results')
```

Comment:
Are you sure that you need "create_stats" attribute running longevity?

Reply:
Not needed as stats are should be not created. Removed

Reply:
Where removed? I see it stayed in the latest patch-set

Reply:
Actually the 'create_stats' has to be initiated. In case of this test with will be False (default value